frontend/catalyst/qfunc.py

```diff
@@ -263,6 +267,37 @@ def wrap_single_shot_qnode(*_):
         results = catalyst.vmap(wrap_single_shot_qnode)(arg_vmap)
         if isinstance(results[0], tuple) and len(results) == 1:
             results = results[0]
-        return parse_native_mid_circuit_measurements(cpy_tape, aux_tapes, results, interface="jax")
```

Comment:
we have a `make format` (from the top level directory where the usual makefile is) that very handily solves these "line-too-long" formatting issues